### ISSUE TITLE
Russians can't play on /tg/

### ISSUE THREAD

**Issue by Traysoul** [tgstation/tgstation#55482]
Moscow City Court has blocked cloudflare ip in Russia. Because of this, Russians cannot play on /tg/ servers.
[https://isitblockedinrussia.com/?host=https%3A%2F%2Fwww.tgstation13.download%2Fbyond%2Fassets%2Fac%2Fasset.aca0e969fcef5f715bbee058633a666c.css](url)
[https://mos-gorsud.ru/mgs/defend/statementForm/9d01e3d5-025e-44bd-9c1b-ea3521c8366a](url)
![Screenshot_15](https://user-images.githubusercontent.com/58553282/101987436-23091400-3ca5-11eb-96bc-f8475b8133ac.png)

**Comment by nullAdept** [comment]
it's true, we can't..

**Comment by Iamgoofball** [comment]
We're going to need to figure this out quick, as the only solution right now is illegal.

**Comment by Traysoul** [comment]
Alright Booktower fixed that. Thanks to him.

**Comment by Iamgoofball** [comment]
This hasn't actually been fixed, a workaround was discovered but we need to fix the actual issue. Can someone re-open this?

**Comment by optimumtact** [comment]
There is absolutely nothing we can do to fix this issue.

**Comment by stylemistake** [comment]
We can resolve DNS on the server, and send backup CDN urls with plain IPs, and once tgui detects tampered content, it can try loading from the fallback url. This should prevent DNS spoofing.

But I don't want to code it.

**Comment by MrStonedOne** [comment]
The way I have it setup right now is `tgstation13.download` goes direct, and `www.tgstation13.download` goes to cloudflare. i set the asset webroot to the apex domain for now.

I think the best path would be to make a redirector endpoint that just sends the user to the www. domain or the apex domain depending on where they are.

This might seem like something that is objectively worse then doing it in game code side, but then an issue comes up with the cdn rsc, and how we have to set that in client/New(), any ip country location stuff wouldn't work unless we wanted to block client/New() until the request finishes.

The webserver has a geoip plugin and provides that info as a http header to php.

**Comment by AlinaStarkova** [comment]
huh, so thats what it was, i used to crash a lot a few weeks ago, like 3 times a round, getting that error. not russian though. i havent had any issues with it recently however

**Comment by optimumtact** [comment]
then this is entirely unrelated to your own issue

**Comment by Iamgoofball** [comment]
This appears to be cropping up again.

**Comment by MrStonedOne** [comment]
i had to enable the cloudflare shit on the cdn again because of ddos. cloudflare is told aggressively cache things in that folder so the hope is it can help ease cdn issues when the website is down.

**Comment by Iamgoofball** [comment]
Looks like this specific issue isn't cropping up, there's instead another issue w/ cloudflare, but a user found a workaround.

**Comment by jlsnow301** [comment]
via discord: russia has had cloudfare unblocked for some time. feel free to reopen this issue if that's not the case

![image](https://github.com/tgstation/tgstation/assets/42397676/695f2a29-9a09-4e8b-9e10-18e6eeb94684)
